client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-details.jsx

```diff
@@ -7,18 +7,18 @@ var React = require( 'react' );
  * Internal dependencies
  */
 var i18n = require( 'lib/mixins/i18n' ),
+	support = require( 'lib/support' ),
 	analyticsMixin = require( 'lib/mixins/analytics' );
 
 var EmailForwardingDetails = React.createClass( {
 	mixins: [ analyticsMixin( 'domainManagement', 'emailForwarding' ) ],
 
 	render: function() {
-		var supportLink = 'https://' + i18n.getLocaleSlug() + '.support.wordpress.com/email-forwarding/';
```

Comment:
Maybe we should add `i18n.getLocaleSlug()` like this to the root support URL in `client/lib/support/index.js`? I honestly don't know what's the best way to proceed here, just making a comment so that someone more knowledgeable might be able to answer :pensive: